index.bs

```diff
@@ -706,8 +710,9 @@ The <dfn method for=CookieStore>set(|options|)</dfn> method steps are:
         |options|["{{CookieInit/value}}"],
         |options|["{{CookieInit/expires}}"],
         |options|["{{CookieInit/domain}}"],
-        |options|["{{CookieInit/path}}"], and
-        |options|["{{CookieInit/sameSite}}"].
+        |options|["{{CookieInit/path}}"],
+        |options|["{{CookieInit/sameSite}}"], and
+        |partitioned|.
```

Comment:
... and this can just be `|options|["{{CookieStoreDeleteOptions/partitioned}}"]` now.

Reply:
Done.